src/main/java/coupon/domain/MinimumOrderAmount.java

```diff
@@ -0,0 +1,41 @@
+package coupon.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Embeddable
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class MinimumOrderAmount {
+
+    private static final int MIN_ORDER_AMOUNT = 5000;
+    private static final int MAX_ORDER_AMOUNT = 100000;
+
+    @Column(name = "minimum_order_amount", nullable = false)
+    private int amount;
+
+    public MinimumOrderAmount(int amount) {
+        validateAmount(amount);
+        this.amount = amount;
+    }
+
+    private void validateAmount(int amount) {
+        validateMinimumAmount(amount);
+        validateMaximumAmount(amount);
+    }
```

Comment:
TMI하자면 ㅎㅎ 개인적으로 저는 validate 해야할 내용이 간단한 경우에는 메서드 분리하지 않는 편이에요 !
정말 사람 바이 사람이겠지만 저는 이런 경우는 분리하지 않았을 때 더 가독성이 좋다고 느낍니다 ~!

```suggestion
    private void validateAmount(int amount) {
        if (amount < MIN_ORDER_AMOUNT) {
            throw new IllegalArgumentException("최소 주문 금액은 " + MIN_ORDER_AMOUNT + "원 이상이어야 합니다.");
        }
        if (amount > MAX_ORDER_AMOUNT) {
            throw new IllegalArgumentException("최소 주문 금액은 " + MAX_ORDER_AMOUNT + "원 이하여야 합니다.");
        }
    }
```

Reply:
오 좋은데요 👍 참고해서 반영하겠습니다.